### PR TITLE
Modify Build

### PR DESCRIPTION
**Pull request by davidfloegel**
- [x] Remove Fonts
- [x] Update dependencies
- [x] Remove unused dependencies

### CODE REVIEW

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/196
Branch:
https://sofarsounds.github.io/maestro/branches/remove-fonts
Storybook:
https://sofarsounds.github.io/maestro/8ba74fdae933676042f867cd2402aef1657585b6/
Build:
https://sofarsounds.github.io/maestro/8ba74fdae933676042f867cd2402aef1657585b6/dist.zip

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/196
Branch:
https://sofarsounds.github.io/maestro/branches/remove-fonts
Storybook:
https://sofarsounds.github.io/maestro/a86822df5c589e6e51310654f457925933ed452a/
Build:
https://sofarsounds.github.io/maestro/a86822df5c589e6e51310654f457925933ed452a/dist.zip

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/196
Branch:
https://sofarsounds.github.io/maestro/branches/remove-fonts
Storybook:
https://sofarsounds.github.io/maestro/a86822df5c589e6e51310654f457925933ed452a/
Build:
https://sofarsounds.github.io/maestro/a86822df5c589e6e51310654f457925933ed452a/dist.zip

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/196
Branch:
https://sofarsounds.github.io/maestro/branches/remove-fonts
Storybook:
https://sofarsounds.github.io/maestro/28b1210d147509c4806b15a6dfada0345626e7d9/
Build:
https://sofarsounds.github.io/maestro/28b1210d147509c4806b15a6dfada0345626e7d9/dist.zip

File: src/molecules/Dropdown/__snapshots__/Dropdown.test.tsx.snap [hunk 1]
```diff
@@ -179,7 +179,7 @@ exports[`<Dropdown /> renders correctly 1`] = `
             "attrs": Array [],
             "componentStyle": ComponentStyle {
               "componentId": "sc-htpNat",
-              "isStatic": true,
+              "isStatic": false,
```
Comment (liondedan) [inline src/molecules/Dropdown/__snapshots__/Dropdown.test.tsx.snap]: Why have these values changes?

**Pull request merged**